pkgs/development/libraries/unixODBCDrivers/default.nix

```diff
@@ -180,6 +180,115 @@
     };
   };
 
+  msodbcsql18 =
+    if stdenv.targetPlatform.isDarwin
```

Comment:
Could you try to replace this `if-else` with an expression common for both platforms? Many attributes, (especially `meta`) is common to all platforms...

Reply:
@doronbehar, I can think of a few ways to remove the duplication. What would you prefer?

1. Add a `if isDarwin then … else …` block for each attribute. There are 5 attributes that would need this. (FWIW I started doing this and it seems to result in _more_ duplication.)
2. Pull out two functions that each takes the common data and then adds OS-specific attributes, then call the correct one.
3. Keep the variants separate but move them to top-level attributes (e.g. `msodbcsql18Linux`) and then just alias the correct one to `msodbcsql18`.

There might be a better way that I am not aware of.

Reply:
> 1. ... FWIW I started doing this and it seems to result in _more_ duplication.

Sounds surprising... Sparing the `meta` duplication seems to win this IMO.

> 2. ...

That's too complicated in my opinion.

> 3. ...

No no no.

Reply:
Alright, done. I also rebased and squashed.